_posts/2019-08-01-CocoaPods-1.8.0-beta.markdown

```diff
@@ -0,0 +1,203 @@
+---
+layout: post
+title:  "CocoaPods 1.8 Beta is Here!"
+author: dnkoutso
+categories: cocoapods releases
+---
+
+_CocoaPods 1.8_ switches the CDN as the default spec repo source and comes with a few enhancements!
+
+<!-- more -->
+
+This is an exciting release as the [master specs repo](https://github.com/CocoaPods/Specs) is no longer required to be cloned to start using CocoaPods!
+
+### CDN as Default
+
+CDN support was first introduced in the [1.7 release](https://blog.cocoapods.org/CocoaPods-1.7.0-beta/) and aims to speed up initial setup and dependency analysis dramatically. With 1.8, CocoaPods no longer requires cloning the now huge [master specs repo](https://github.com/CocoaPods/Specs) in order to function.
+
+You may safely delete the [master specs repo](https://github.com/CocoaPods/Specs) with the following steps:
+
+First, edit your `Podfile` to set the CDN as the primary source:
+
+```diff
+- source 'https://github.com/CocoaPods/Specs.git'
++ source 'https://cdn.cocoapods.org/'
+```
+
+Afterwards, run the following command to remove the [master specs repo](https://github.com/CocoaPods/Specs):
+
+```
+pod repo remove master
+```
+
+If you wish to continue using the `git` based source there is no action to be taken and CocoaPods should continue working as usual.
+
+Upon installing CocoaPods for the first time, the `git` based [master specs repo](https://github.com/CocoaPods/Specs) will _no longer_ be cloned and users may start using CocoaPods almost instantly. Below you can find a video demonstration of integrating a project with CocoaPods 1.8:
```

Comment:
Maybe add a note that the CDN will also be used if no source is specified in the Podfile

Reply:
done